libraries/eosiolib/contracts/eosio/map.hpp

```diff
@@ -498,6 +499,19 @@ namespace eosio::kv {
 
          friend iterator_t;
 
+         static detail::packed_view get_tmp_buffer(std::size_t size_needed=0) {
```

Comment:
moving this helped `::materialize` compile to the next error which only occurs when using accessing `const` iterator:

```
/include/eosiolib/contracts/eosio/map.hpp:262:13: error: 'this' argument to member function 'materialize' has type 'const eosio::kv::detail::iterator<false, eosio::kv::map<...> >', but function is not marked const
            materialize();
            ^~~~~~~~~~~
```